module_utils/network_lsr/argument_validator.py

```diff
@@ -246,6 +247,62 @@ def _validate_string_min_length(self, value):
             return True
 
 
+class ArgValidatorRouteTable(ArgValidator):
+    def __init__(
+        self,
+        name,
+        required=False,
+        default_value=None,
+    ):
+        ArgValidator.__init__(self, name, required, default_value)
+
+    def _validate_impl(self, value, name):
+        norm_value = None
+        try:
+            if isinstance(value, bool):
+                # bool can (probably) be converted to self.numeric_type,
+                # but here we don't want to accept a boolean value.
+                pass
+            elif isinstance(value, int):
+                norm_value = int(value)
+            elif isinstance(value, Util.STRING_TYPE):
+                try:
+                    norm_value = int(value)
+                except Exception:
+                    norm_value = value
+        except Exception:
+            pass
+        if norm_value is None:
+            raise ValidationError(
+                name,
+                "route table must be the named or numeric tables but is '%s'" % (value),
```

Comment:
In this exception message, the `%s` is embedded inside single quotes, in the other messages it is not. It would be good to make this uniform. Also, for new code it seems to be best to use ".format()" which is already used in some areas.